src/main/java/duke/Command.java

```diff
@@ -0,0 +1,8 @@
+package duke;
+
+/**
+ * Command interface to be passed onto Parser.
+ */
+interface Command {
+    void execute(String string) throws DukeException;
```

Comment:
Perhaps you can write a javadoc comment for this execute method and {@inherit} the overriding methods?

Reply:
Similarly, javadoc headers are missing for most public classes/methods. I believe those should be completed.